openapi-spec/swagger.yaml

```diff
@@ -1997,78 +2052,104 @@ components:
           example: '2017-07-10T14:36:58.014Z'
           readOnly: true
     DataStorageLoS:
-      description: >-
-        DataStorageLoS can be used to describe a service option covering storage
-        provisioning and availability.
       type: object
       properties:
         recoveryTimeObjective:
+          example: 10
+          description: Recovery Time Objective(RTO) is a maximum acceptable time within which storage must be restored after disaster. For e.g 10 sec.
           type: integer
           format: int64
         provisioningPolicy:
+          default: thick
+          description: The provisioning policy of the pool. It can be thick or thin.
           type: string
           enum:
             - Thin
             - Thick
         compression:
+          description: The optimization policy to remove redundant data within the file
```

Comment:
Compression is not removing redundant data but reduction in the number of bits in the data to save storage

Reply:
done